kubectl.ts

```diff
@@ -31,26 +31,43 @@ export class KubectlCommand {
         }   
         this.kubectlbinary = tl.getInput('kubectlBinary');
         this.configfile = path.join(tl.cwd(), "config");
-        this.kubectl = tl.tool(this.kubectlbinary);
     }
     append(arg) {
         this.kubectl.arg(arg);
     }
     exec() {
         this.execCommand();
     }
+    async init() {
+        if (!fs.exists(this.kubectlbinary)) {                
+            tl.debug("downloading kubectl");
+            let curl: ToolRunner = tl.tool("curl");
+            curl.arg("-LO").arg("https://storage.googleapis.com/kubernetes-release/release/v1.5.2/bin/linux/amd64/kubectl");
```

Comment:
Only the problem is the version number. v1.5.2. This feature has backward compatibility. So currently no problem. However, in the future, we need to add the version column on the Task as optional. And it is not difficult. because we can just insert the version number. https://kubernetes.io/docs/user-guide/prereqs/